src/test/scala/shine/OpenCL/Barriers.scala

```diff
@@ -0,0 +1,170 @@
+package shine.OpenCL
+
+import util.gen
+import rise.core.DSL._
+import rise.core.DSL.Type._
+import rise.core.types._
+import rise.core.types.AddressSpace._
+import rise.core.primitives._
+import rise.openCL.primitives.oclReduceSeq
+import rise.openCL.TypedDSL._
+
+// scalastyle:off org.scalastyle.scalariform.MultipleStringLiteralsChecker
+class Barriers extends test_util.Tests {
+  private val sum = oclReduceSeq(Private)(add)(l(0.0f))
+
+  // TODO?
+  ignore("1D mapLocal toLocal seq without thread sharing") {
```

Comment:
Could you add a comment here on why these tests are currently ignored? (like you have it below)

Reply:
done